common/metrics/metrics.go

```diff
@@ -58,6 +59,10 @@ type (
 		Histogram(string, MetricUnit) HistogramIface
 
 		Stop(log.Logger)
+
+		// BatchStart returns a Handler that where supported, can emit a series of metrics as a single "wide event".
+		// If wide events aren't supported in the underlying implementation, metrics can still be sent individually.
+		BatchStart(string) (Handler, io.Closer)
```

Comment:
Can you make this return a BatchHandler where that interface extends Handler and io.Closer?

Reply:
Also think StartBatch reads better than BatchStart since batch is also a verb.

Reply:
Done